python/lsst/ts/scheduler/scheduler_csc.py

```diff
@@ -806,139 +848,52 @@ async def configure(self, config):
         else:
             self.log.warning("No 'dependenciesVersions' event.")
 
-        await self.evt_obsSiteConfig.set_write(
-            observatoryName=settings.models["location"]["obs_site"]["name"],
-            latitude=settings.models["location"]["obs_site"]["latitude"],
-            longitude=settings.models["location"]["obs_site"]["longitude"],
-            height=settings.models["location"]["obs_site"]["height"],
-        )
+        await self._publish_settings(settings)
 
-        await self.evt_telescopeConfig.set_write(
-            altitudeMinpos=settings.models["observatory_model"]["telescope"][
-                "altitude_minpos"
-            ],
-            altitudeMaxpos=settings.models["observatory_model"]["telescope"][
-                "altitude_maxpos"
-            ],
-            azimuthMinpos=settings.models["observatory_model"]["telescope"][
-                "azimuth_minpos"
-            ],
-            azimuthMaxpos=settings.models["observatory_model"]["telescope"][
-                "azimuth_maxpos"
-            ],
-            altitudeMaxspeed=settings.models["observatory_model"]["telescope"][
-                "altitude_maxspeed"
-            ],
-            altitudeAccel=settings.models["observatory_model"]["telescope"][
-                "altitude_accel"
-            ],
-            altitudeDecel=settings.models["observatory_model"]["telescope"][
-                "altitude_decel"
-            ],
-            azimuthMaxspeed=settings.models["observatory_model"]["telescope"][
-                "azimuth_maxspeed"
-            ],
-            azimuthAccel=settings.models["observatory_model"]["telescope"][
-                "azimuth_accel"
-            ],
-            azimuthDecel=settings.models["observatory_model"]["telescope"][
-                "azimuth_decel"
-            ],
-            settleTime=settings.models["observatory_model"]["telescope"]["settle_time"],
-        )
+    async def validate_observing_blocks(self) -> None:
+        """Validate observing blocks.
 
-        await self.evt_rotatorConfig.set_write(
-            positionMin=settings.models["observatory_model"]["rotator"]["minpos"],
-            positionMax=settings.models["observatory_model"]["rotator"]["maxpos"],
-            positionFilterChange=settings.models["observatory_model"]["rotator"][
-                "filter_change_pos"
-            ],
-            speedMax=settings.models["observatory_model"]["rotator"]["maxspeed"],
-            accel=settings.models["observatory_model"]["rotator"]["accel"],
-            decel=settings.models["observatory_model"]["rotator"]["decel"],
-            followSky=settings.models["observatory_model"]["rotator"]["follow_sky"],
-            resumeAngle=settings.models["observatory_model"]["rotator"]["resume_angle"],
-        )
+        Notes
+        -----
+        This method will walk through the observing blocks defined in the model
```

Comment:
Please use an active voice so `This method will walk through the observing blocks` -> `Walk through the observing blocks`